src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java

```diff
@@ -238,4 +267,17 @@ private ConditionalOrderCourtDetails setupConditionalOrderCourtDetailsConfig() {
         when(conditionalOrderCourtDetailsConfig.get(BURY_ST_EDMUNDS.getCourtId())).thenReturn(expectedDetails);
         return expectedDetails;
     }
+
+    private CtscContactDetails buildCtscContactDetails() {
```

Comment:
We should add these values to DocmosisTemplateConstants so they can be utilised properly.